hipchat.go

```diff
@@ -201,9 +204,14 @@ func (c *Client) listen() {
 				continue
 			}
 
+			// parse time
+			time_pieces := strings.Split(attr["ts"], ".")
```

Comment:
totally up to you; i only care about the attribute. i'm writing a bot and
don't want to get flooded with stale commands on start-up.

On Thu, Jan 22, 2015 at 1:33 PM, Dane Harrigan <notifications@github.com>
wrote:

> In hipchat.go
> <https://github.com/daneharrigan/hipchat/pull/5#discussion_r23402713>:
>
> > @@ -201,9 +204,14 @@ func (c *Client) listen() {
> >  				continue
> >  			}
> >
> > +			// parse time
> > +			time_pieces := strings.Split(attr["ts"], ".")
>
> can time_pieces be renamed to ts
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/daneharrigan/hipchat/pull/5/files#r23402713>.
>



-- 
taw